### DISCUSSION TITLE
Create tag "version 1.0" for BLOM?

### DISCUSSION THREAD

**Discussion by TomasTorsvik** [NorESMhub/BLOM#249] [Team Posts]
Hi,
I'm thinking it could make sense to create a tag "version 1.0" (or something similar) for BLOM already now. Now that the licensing and copyright issue has been settled, I think it can be good to have a "official" starting point for the new BLOM development. Also, it can be a useful test to see how the workflow we discussed earlier will work in practice. What do you think?
Best wishes,
Tomas

**Comment by matsbn** [comment]
I would prefer to wait with version 1.0 or similar until we have renamed 'MICOM' to 'BLOM' in the interface with NorESM and in NorESM itself. Right now we are in a time-squeeze to get a DOI'ed archive for the GMD publications of Jerry and Øyvind. I want to update the very initial tag 'blom0_0_000' to the current master that includes copyright and license so we do not have any tag's that point to code without it.
